GameServerLib/GameObjects/AttackableUnits/AI/BaseTurret.cs

```diff
@@ -23,13 +23,14 @@ public class BaseTurret : ObjAiBase, IBaseTurret
             float y = 0,
             TeamId team = TeamId.TEAM_BLUE,
             uint netId = 0
-        ) : base(game, model, new Stats.Stats(), 50, x, y, 1200, netId)
+        ) : base(game, model, new Stats.Stats(), 100, x, y, 1200, netId)
```

Comment:
Reason behind this change? It's not in the description.

Reply:
The 50 was wrong and it seems that turrets have 120 or so. I wasn't sure and I think we should get it from the json with some map rework (should add `TODO: find out the right number`)